app/views/course/levels/index.html.slim

```diff
@@ -1,8 +1,14 @@
 h1 = t('.header')
+
+- if can?(:create, Course::Level)
+  = new_button([current_course, :level]) { t('.create') }
```

Comment:
Leave it as an icon first, because that's what it is across the entire site. Maybe I'll change that to text in future, but for now, follow convention unless you have a compelling reason.